### ISSUE TITLE
Firefox 82.0.2 blank window: An unexpected property was found in the WebExtension manifest

### ISSUE THREAD

**Issue by jmahhh** [MetaMask/metamask-extension#9756]
Firefox Browser, Windows, 82.0.2 (64-bit)

Metamask 8.1.2

Extension list says: 
![image](https://user-images.githubusercontent.com/13865534/97591207-4a5a9900-19f7-11eb-87f3-8c2a5073835f.png)

Inspect says:
![image](https://user-images.githubusercontent.com/13865534/97591269-58101e80-19f7-11eb-9344-45f5bada1004.png)

Manifest section:
```
  "externally_connectable": {
    "matches": [
      "https://metamask.io/*"
    ],
    "ids": [
      "*"
    ]
  },
```

**Comment by tmashuang** [comment]
What were you doing before the blank screen occurred? None of the screen shots provided has information that can help your issue.

**Comment by jmahhh** [comment]
@tmashuang nothing special related to metamask that I can remember.

**Comment by Incedis** [comment]
Exact same issue as far as the symptoms and it started to happen when Firefox updated to 82.0.2. Any help is much appreciated.

**Comment by Gudahtt** [comment]
The `externally_connectable` warning is a long-standing issue that has no user impact. If you are seeing a blank screen, it is likely unrelated to everything you have shown here.

**Comment by Gudahtt** [comment]
The blank screen issue is being tracked here: https://github.com/MetaMask/metamask-extension/issues/9196#issuecomment-722014269
Please contact support if you encounter this again for further instructions! I'd love to get more information about how this might happen.

I'll leave this issue open for now to track the `externally_connectable` warning.

**Comment by Gudahtt** [comment]
The warning shown here will be fixed by #9804
